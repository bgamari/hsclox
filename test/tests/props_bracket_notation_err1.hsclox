class B {}
var b = B();
print b[1];
