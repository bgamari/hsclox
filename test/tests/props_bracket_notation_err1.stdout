Only strings allowed as fields.
[line 3] in script
